src/components/Calendar/Calendar.tsx

```diff
@@ -190,6 +192,29 @@ const makeAllDayEventsInclusive = (endDateExclusiveEvents: CalendarEvent[]): Cal
     });
 };
 
+const initialLocations = [
+    "Vauxhall Hope Church",
+    "Waterloo - St George the Martyr",
+    "Waterloo - Oasis",
+    "Waterloo - St Johns",
+    "Brixton Hill - Methodist Church",
+    "N&B - Emmanuel Church",
+    "Streatham - Immanuel and St Andrew",
+];
```

Comment:
It hasn't been merged yet right? Or has it?

Reply:
Also isn't generalising the database stuff a separate ticket? Will add a todo, ofc if that is merged I will change, just thinking we will want to fetch that data once not repeatedly everywhere!

Reply:
The collection_centres database already exist so you have the ability to implement it
My current ticket is to fix existing implementation of collection centres, but since your function is a new feature you are implementing, I think this will be more appropriate to implement it on your side.
Since my side has no access to your current implementation it would make sense for your side to implement it I think

Reply:
Especially since you are the one implementing it you probably know better how to extend it to general collection centres better than I do